test/views/EditRecordViewSpec.scala

```diff
@@ -48,13 +47,16 @@ class EditRecordViewSpec extends BaseSpec {
           formerReferenceDepartment = "TestFormerReferenceDepartment",
           startDate = "1/1/1960",
           endDate = "31/12/1960"
-        ))
+        )
+      )
 
       document must haveTitle("TNA reference: COAL 80/80/1")
-      document must haveHeading("PAC-ID: COAL.2022.V5RJW.P")
+      document must haveHeading("edit-set.record.edit.heading")
```

Comment:
The messages instance is empty because it is stubbed here, because we are using messages now instead of passing CCR explicitly the value for the header is just the messages key.

Reply:
Also the old assertion was wrong, CCR should be using in the `<h1>`, not the OCI.

Reply:
Is there a flaw in `haveHeading`?